data/Hints/league.json

```diff
@@ -1,31 +1,46 @@
 {
-    "name":                  "league",
-    "gui_name":              "League S3",
-    "description":           "Hint Distribution for the S3 of League. 5 Goal Hints, 3 Barren Hints, 5 Sometimes hints, 7 Always hints (including skull mask), Several hints removed from the Sometimes hint pool.",
+    "name":                  "leagueS4",
```

Comment:
This field must match the filename. We also use snake case convention here. Either change both this field and the filename to `league_s4` or change this field back to `league`. I recommend the latter since it avoids breaking settings caches.

Reply:
Went with the later in this case and pushed the update